diff --git a/node_modules/util/util.js b/node_modules/util/util.js
index 6db393e..c15bfbc 100644
--- a/node_modules/util/util.js
+++ b/node_modules/util/util.js
@@ -106,7 +106,7 @@ exports.deprecate = function(fn, msg) {
 var debugs = {};
 var debugEnvRegex = /^$/;
 
-if (process.env.NODE_DEBUG) {
+if (typeof process !== 'undefined' && process.env.NODE_DEBUG) {
   var debugEnv = process.env.NODE_DEBUG;
   debugEnv = debugEnv.replace(/[|\\{}()[\]^$+?.]/g, '\\$&')
     .replace(/\*/g, '.*')
@@ -354,11 +354,13 @@ function formatPrimitive(ctx, value) {
   // For some reason typeof null is "object", so special case here.
   if (isNull(value))
     return ctx.stylize('null', 'null');
+  if (isSymbol(value))
+    return ctx.stylize(value.toString(), 'symbol');
 }
 
 
 function formatError(value) {
-  return '[' + Error.prototype.toString.call(value) + ']';
+  return value.toString() + '\n' + value.stack;
 }
 
 
