diff --git a/node_modules/util/util.js b/node_modules/util/util.js
index 6db393e..a2d8cd4 100644
--- a/node_modules/util/util.js
+++ b/node_modules/util/util.js
@@ -106,7 +106,7 @@ exports.deprecate = function(fn, msg) {
 var debugs = {};
 var debugEnvRegex = /^$/;
 
-if (process.env.NODE_DEBUG) {
+if (typeof process !== 'undefined' && process.env.NODE_DEBUG) {
   var debugEnv = process.env.NODE_DEBUG;
   debugEnv = debugEnv.replace(/[|\\{}()[\]^$+?.]/g, '\\$&')
     .replace(/\*/g, '.*')
@@ -358,7 +358,7 @@ function formatPrimitive(ctx, value) {
 
 
 function formatError(value) {
-  return '[' + Error.prototype.toString.call(value) + ']';
+  return value.toString() + '\n' + value.stack;
 }
 
 
